data/language/dutch.txt

```diff
@@ -1709,7 +1709,7 @@ STR_1707    :Teveel werknemers in dit park
 STR_1708    :{SMALLFONT}{BLACK}Werkgebied van deze werknemer instellen
 STR_1709    :Werknemer ontslaan
 STR_1710    :Ja
-STR_1711    :{WINDOW_COLOUR_1}Weet je zeker dat je {STRINGID} wilt ontslaan?
+STR_1711    :{WINDOW_COLOUR_1}Weet je zeker dat je {STRINGID} wil ontslaan?
```

Comment:
Je ziet de laatste tijd steeds meer het gebruik van "jij wil", en denk wilt langzamerhand uit het Nederlands aan het verdwijnen is. Het is bijvoorbeeld de enige vorm in het hele rijtje met een t erachter, en ook bij "je" in de betekenis van "men" ("Hier vind je alles wat je wil weten").
Ik denk dat het voor de consistentie en de toegankelijkheid beter is de iets informele versie te gebruiken.